compute_tools/src/bin/fast_import.rs

```diff
@@ -310,6 +340,12 @@ pub(crate) async fn main() -> anyhow::Result<()> {
         }
     }
 
+    // If interactive mode, wait for Ctrl+C
+    if args.interactive {
+        info!("Running in interactive mode. Press Ctrl+C to shut down.");
+        tokio::signal::ctrl_c().await.context("wait for ctrl-c")?;
+    }
```

Comment:
I'm not entirely sure what the intention is here. Is the idea to keep the PG we restored in running? If so, why not just start it up again after the program exits instead of having this extra flag?

Reply:
it is just a convenience for local testing, when I debug pgdump/restore results, it is easy to run in interactive and look into the db contents